uint8 id1
uint8 id2
uint8 id3
int32 velocity1
int32 velocity2
int32 velocity3
